Sources/SwiftGui/SwiftGui.swift

```diff
@@ -14,24 +14,188 @@ public class MyApplication: Application {
     super.init(name: "org.swift.Foo")
   }
 
-  public override func onActivate() {
+  func helloButton() -> Button {
+    Button("Hello!").handler {
+      print("Hello!")
+    }
+  }
+
+  func aboutWindow(win: Window) -> AboutWindow {
+    AboutWindow(app: self, parent: win)
+      .generalData(title: "SwiftGui", icon: .default(icon: .applicationXExecutable), developer: "JCWasmx86", version: "main")
+      .website(url: "https://github.com/JCWasmx86/SwiftGui")
+      .issue(url: "https://github.com/JCWasmx86/SwiftGui/issues")
+      .developers(["JCWasmx86 https://github.com/JCWasmx86", "david-swift https://github.com/david-swift"])
+      .copyright("© 2023 JCWasmx86")
```

Comment:
Here you can add yourself too, as you wrote more code than I did ^^ (BTW, should I make you a maintainer? SO you can instantly push to main?)